webserver/views/api/legacy.py

```diff
@@ -42,10 +42,13 @@ def get_high_level(mbid):
     If there is more than one document with the same mbid, you can specify
     an offset as a query parameter in the form of ?n=x, where x is an integer
     starting from 0
+    If you need the full key values instead of the shortened keys, you can specify
+    a parameter 'map=True' where map is a boolean value
     """
     mbid, offset = _validate_data_arguments(mbid, request.args.get("n"))
+    map_keys = request.args.get("map")
```

Comment:
We should make the parameter name more explicit. How about `map_model_classe_names`?
The documentation could be updated to be a bit clearer:
> Some models use short codes as class names. To get more descriptive class names for these models, set a query parameter `map_model_classe_names=True`.

Reply:
i wanted to keep the parameter name concise as it has to be passed with the api url manually, dont you think?

Reply:
i thought it would be better to keep the parameter name short as it has to be passed with the api url, should i just change the documentation and keep the name as is?

Reply:
I prefer to have the parameter easy to understand, rather than short. Can we name it `map_model_class_names`?

Reply:
sure, i'll rename it :)